### ISSUE TITLE
Text overwritten in pipresents editor

### ISSUE THREAD

**Issue by ian2048** [KenT2/pipresents-beep#59]
Hi, when the editor is running on a browser (Chrome) I find that any profiles or track names that are too long wrap onto the next line and overwrite the next entry - I guess they should really move all the text below down a line to make space. This is not critical but just a bit irritating, trying to read one line of text through another.

**Comment by KenT2** [comment]
Yes I know that can be a problem. If I had bigger fields I could not fit everything on a reasonable sized screen.
